app/views/spree/products/_properties.html.erb

```diff
@@ -1,13 +1,23 @@
-<% unless @product_properties.blank? %>
-  <h3 class="pt-4 font-weight-bold text-uppercase product-details-subtitle"><%= Spree.t(:details) %></h3>
+<%# unless @product_properties.blank? %>
+  <!-- <h3 class="pt-4 font-weight-bold text-uppercase product-details-subtitle"><%#= Spree.t(:details) %></h3>
   <ul id="product-properties" class="m-0 list-unstyled product-properies" data-hook>
-    <% @product_properties.each do |product_property| %>
-      <% if product_property.show_property? %>
+    <%# @product_properties.each do |product_property| %>
+      <%# if product_property.show_property? %>
         <li>
-          <span class="font-weight-bold text-break"><%= product_property.property.presentation %>:</span>
-          <span class="text-break"><%= product_property.value %></span>
+          <span class="font-weight-bold text-break"><%#= product_property.property.presentation %>:</span>
+          <span class="text-break"><%#= product_property.value %></span>
         </li>
-      <% end %>
-    <% end %>
-  </ul>
-<% end %>
+      <%# end %>
+    <%# end %>
+  </ul> -->
+<%# end %>
+<div class="product-ingredients" id="product-properties"  data-hook data-hook="product_properties">
+    <h5>Ingredients</h5>
+    <% unless @product_properties.blank? %>
+        <% @product_properties.each do |product_property| %>
+          <% if product_property.show_property? %>
+              <span ><%= product_property.value %></span>
```

Comment:
Please show the property presentation as well with value. As its already displaying in spree default.

<%= product_property.property.presentation %>
<img width="879" alt="Screen Shot 2021-12-09 at 1 59 57 AM" src="https://user-images.githubusercontent.com/77246192/145283393-2ff6ef23-4c47-42c9-a7c1-88f1528d0186.png">

Reply:
sir in our template we dont need presentation